nilearn/datasets/atlas.py

```diff
@@ -0,0 +1,577 @@
+# *- encoding: utf-8 -*-
```

Comment:
there are special characters in the file that is why python complained without this line